aesm-client/Cargo.toml

```diff
@@ -28,6 +28,7 @@ test-sgx = []
 # Project dependencies
 sgxs = { version = "0.7.0", path = "../sgxs", optional = true }
 sgx-isa = { version = "0.3.0", path = "../sgx-isa"}
+sgx_types = { version = "1.1.1" }
```

Comment:
This is in the wrong section. This is an external dependency.

Reply:
thanks